physionet-django/environment/templates/environment/workspace_being_provisioned.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+
+{% load static %}
+
+{% load project_templatetags %}
+
+{% block title %}Research environments{% endblock %}
+
+{% block local_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'environment/css/environment-base.css' %}">
+{% endblock %}
+
+{% block local_js_top %}
+<script src="{% static 'custom/js/cookie.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div class="container">
+  {% include "message_snippet.html" %}
+  <div class="jumbotron">
+    <h1>Research Environments Home</h1>
+    <p class="lead">
+      Here you can manage the research environments for your projects.
+    </p>
+  </div>
+    <div class="card text-center" style="width: 30rem; margin: auto">
+      <div class="card-body">
+        <h5 class="card-title">Please wait</h5>
+        <p class="card-text">Your workspace is being provisioned...</p>
+          <div class="loader">Loading...</div>
+      </div>
+    </div>
+</div>
+{% endblock %}
+
+{% block local_js_bottom %}
+<script>
+    const csrfToken = getCookie("csrftoken");
+    let workspaceInterval = setInterval(check_workspace_status, 15000);
```

Comment:
```suggestion
    const workspaceInterval = setInterval(check_workspace_status, 15000);
```
I don't see it reassigned anywhere